certification/internal/shell/scorecard_basic-check-spec-test.go

```diff
@@ -0,0 +1,135 @@
+package shell
```

Comment:
Let's be consistent in filenames. Use all underscores. And don't use '-test' or '_test', as that would normally signify a golang file that contains unit (or some other) test.

Reply:
I removed all references to test.